platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -264,23 +270,38 @@ class HangingProtocolService {
     this.activeProtocolIds = [...protocolId];
   }
 
+  /**
+   * Sets the active study.
+   * This is the study that the hanging protocol will consider active and
+   * may or may not be the study that is being shown by the protocol currently,
+   * for example, a prior view hanging protocol will NOT show the active study
+   * specifically, but will show another study instead.
+   */
+  public setActiveStudy(activeStudy: StudyMetadata): void {
```

Comment:
I prefer it be `public setActiveStudy(studyInstanceUID)` and you create the StudyMetadata yourself. It is weird it relies on an object that can contain `studyDescription`